File: UtilitiesVB/Solution Interfaces/IToDoObjects.vb

Public Interface IToDoObjects
    ReadOnly Property DictPPL As Dictionary(Of String, String)
    ReadOnly Property DictPPL_Filename As String
    Sub DictPPL_Save()
    ReadOnly Property DictRemap As Dictionary(Of String, String)
    ReadOnly Property IDList As IListOfIDs
    Sub IDList_Refresh()
    ReadOnly Property Parent As IApplicationGlobals
    ReadOnly Property ProjInfo As IProjectInfo
    ReadOnly Property ProjInfo_Filename As String
    ReadOnly Property FnameDictRemap As String
    ReadOnly Property FnameIDList As String

End Interface

File: UtilitiesVB/Solution Interfaces/IToDoObjects.cs
using System.Collections.Generic;
using UtilitiesCS;

namespace UtilitiesVB
{

    public interface IToDoObjects
    {
        Dictionary<string, string> DictPPL { get; }
        string DictPPL_Filename { get; }
        void DictPPL_Save();
        Dictionary<string, string> DictRemap { get; }
        ISerializableList<string> CategoryFilters { get; }
        IListOfIDs IDList { get; }
        IApplicationGlobals Parent { get; }
        IProjectInfo ProjInfo { get; }
        string ProjInfo_Filename { get; }
        string FnameDictRemap { get; }
        string FnameIDList { get; }

    }
}